bouncer/shared/utils.ts

```diff
@@ -344,12 +344,16 @@ export async function observeEvent(
   eventQuery?: EventQuery,
   stopObserveEvent?: () => boolean,
   finalized = false,
+  debug = false,
```

Comment:
I know that you added something similar in a few other places already (e.g. `doPerformSwap`), but I wonder if instead of passing parameters for this we should use and enviornment variable instead (similar to RUST_LOG). Then maybe (in a separate PR) we could write a wrapper around console.log to check that env variable. Wouldn't be too difficult to have different levels as well. Alternatively, we could look into using a third party library for this. The way we do it right now is awkward and inconsistent.

Reply:
We could also leverage a lightweight js logging library, which likely does this plus some more potentially useful features

Reply:
That's what I meant by
> Alternatively, we could look into using a third party library for this.

😉

Reply:
oh lol, missed that completely

Reply:
Created a ticket: https://linear.app/chainflip/issue/PRO-1415/use-a-js-logging-library-in-bouncer

Reply:
Looks like the function is getting removed in #4958 anyway.

Reply:
I'll deal with it in that PR